common/config/rush/pnpm-config.json

```diff
@@ -5,12 +5,8 @@
   "globalPeerDependencyRules": {
     "ignoreMissing": [
       "@babel/core",
```

Comment:
where is this being used?

Reply:
when I ran rush update, it threw the missing peer warning in `ui/appui-abstract`